pkg/api/http_server.go

```diff
@@ -363,6 +366,7 @@ func ProvideHTTPServer(opts ServerOptions, cfg *setting.Cfg, routeRegister routi
 		oauthTokenService:            oauthTokenService,
 		statsService:                 statsService,
 		authnService:                 authnService,
+		starApi:                      starApi,
```

Comment:
I think at this point we should be able to remove star.Service as a dependency of the HTTP Server, since star.API injects that on its own.

Reply:
search and navtree are still using star.Service as dependency.